DirectBoot/Application/src/main/java/com/example/android/directboot/alarms/AlarmAdapter.java

```diff
@@ -67,8 +69,8 @@ public AlarmViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     }
 
     @Override
-    public void onBindViewHolder(AlarmViewHolder holder, final int position) {
-        Alarm alarm = mAlarmList.get(position);
+    public void onBindViewHolder(AlarmViewHolder holder,@SuppressLint("RecyclerView")  final int position) {
+        Alarm alarm = mAlarmList.get(holder.getAdapterPosition());
```

Comment:
Is there a reason not to use `position` here? The `position` argument should determine which item in the list to use.

Reply:
Hi jd
https://stackoverflow.com/questions/34942840/lint-error-do-not-treat-position-as-fixed-only-use-immediately/34943270#34943270

Reply:
Yes. you are right, in this context.  The change is intended to suppress a lint error to pass CI checks, though there is no problem with behaviour. 

[StackOverFlow](https://stackoverflow.com/questions/34942840/lint-error-do-not-treat-position-as-fixed-only-use-immediately/34943270#34943270
)
[Official Documentation](https://developer.android.com/reference/android/support/v7/widget/RecyclerView.Adapter.html#onbindviewholder_1)

Thanks

Reply:
sorry it's my misunderstanding... I revert this change.